config/set/downgrade-php73.php

```diff
@@ -3,7 +3,7 @@
 declare(strict_types=1);
 
 use Rector\Downgrade\Rector\LNumber\ChangePhpVersionInPlatformCheckRector;
-use Rector\DowngradePhp73\Rector\FuncCall\SetCookieOptionsArrayToArguments;
+use Rector\DowngradePhp73\Rector\FuncCall\SetCookieOptionsArrayToArgumentsRector;
```

Comment:
I wonder... this name typo should not happen with [`generate` command](https://github.com/rectorphp/rector/blob/master/docs/rector_recipe.md). I recall there was a bug in naming, but I think we fixed it.

How did you create a rule and tests?

Reply:
I created the files by hand.

Reply:
Oh. I use only the generate command for last 2 years. It handles the boring parts and gives more time for coding the `refactor()` logic :)